doc/src/base/base.md

```diff
@@ -378,6 +378,7 @@ Base.Sys.isjsvm
 Base.Sys.loadavg
 Base.Sys.isexecutable
 Base.Sys.username
+Base.Sys.show
```

Comment:
(`Base.Sys.show === Base.show` — they are the same functions even if you added a method.  If you wanted to document a particular method you would put the argument types here, but as I commented above we don't actually need a separate docstring for every method of `show`.
```suggestion
```